code/modules/research/xenobiology/crossbreeding/warping.dm

```diff
@@ -158,29 +158,36 @@ put up a rune with bluespace effects, lots of those runes are fluff or act as a
 	name = "greyspace rune"
 	desc = "Death is merely a setback, anything can be rebuilt given the right components."
 	icon_state = "rune_grey"
-	///extractype is used to remember the type of the extract on the rune
-	var/extractype
+	///extracttype is used to remember the type of the extract on the rune
+	var/extracttype
 	var/req_extracts = 8
 
 /obj/effect/warped_rune/greyspace/examine(mob/user)
 	. = ..()
-	to_chat(user, "<span class='notice'>Requires absorbing [req_extracts] [extractype ? "[extractype] extracts" : "slime extracts"].</span>")
+	to_chat(user, "<span class='notice'>Requires absorbing [req_extracts] [extracttype ? "[extracttype] extracts" : "slime extracts"].</span>")
 
 /obj/effect/warped_rune/greyspace/do_effect(mob/user)
 	for(var/obj/item/slime_extract/extract in rune_turf)
-		if(extract.color_slime == extractype || !extractype) //check if the extract is the first one or of the right color.
-			extractype = extract.color_slime
+		if(extract.color_slime == extracttype || !extracttype) //check if the extract is the first one or of the right color.
+			extracttype = extract.color_slime
 			qdel(extract) //vores the slime extract
 			req_extracts--
 			if(req_extracts <= 0)
-				new /mob/living/simple_animal/slime (rune_turf, extractype) //spawn a slime from the extract's color
+				switch(extracttype)
```

Comment:
You have ignored why there are so many denied PRs that fix this issue.

*SOLVE IT PROPERLY.*

Reply:
Yea this is a lazy solution, and I see what you wanted in the PR after thinking it through.

However:
Currently this call from warped rune calls the set_colour with the argument "lightpink" compared to normal spawn process that calls with "light pink". Without this change half the slimes would be called "lightpink adult slime" and the other natural half "light pink adult slime", this change fixes both of these issues. Another issue would be the slime mutation colors being all set to "lightpink" etc causing them to no longer evolve.

If we want to go the sanitized name in slimes.dmi route (from what I understand you want them to be named like "lightpink adult slime" in this file), the actual name showing up needs to be decided on. A solution would be changing all current instances of "light pink" etc to "lightpink" (warping.dm and wherever else this issue pops up) and just changing the update_name() in slimes.dm to check for these colors, and then change them properly.

Anyways, decide on how you want me to fix, because the solution according to the PR by itself is not enough.
"the solution is thusly to sanitize the color string immediately, and retool icons/mob/slimes.dmi to index icon by sanitized color."
The issue crops up in the first place because of the sanitized name, the current setup is expecting non sanitized names, and an already sanitized name screws it up.

Reply:
Yeah sure whatever